Short-Answer/Algorithms_Answers.md

```diff
@@ -2,14 +2,17 @@
 
 ## Exercise I
 
-a)
+a) # O(n) due to the size of n or would what it be O(log n) due to n being multiplied each iteration
```

Comment:
Great reasoning, however, the answer is O(1). Normally O(n^3), but here we add n^2 to `a` each time